scss/components/_switch.scss

```diff
@@ -208,6 +216,11 @@ $switch-paddle-transition: all 0.25s ease-out !default;
   // <input> element
   .switch-input {
     @include switch-input;
+    // indicate a disabled switch
+    &:disabled ~ .switch-paddle {
+      cursor: $switch-cursor-disabled;
+      opacity: $switch-opacity-disabled;
+    }
```

Comment:
Sorry for not putting that in my first review, but could you move that to `@mixin switch-paddle`, just after the `input:checked` styles ?